src/svymean.jl

```diff
@@ -54,7 +54,7 @@ function svymean(x::Symbol, design::SimpleRandomSample)
         p.se = sqrt.(p.var)
         return p
     end
-    return DataFrame(mean = mean(design.data[!, x]), sem = sem(x, design::SimpleRandomSample))
+    return DataFrame(mean=mean(design.data[!, x]), sem=sem(x, design::SimpleRandomSample))
```

Comment:
Is this the correct style?

Reply:
i ran julia formatter so that should make these lines  consistent